src/components/config/parse-postman.ts

```diff
@@ -35,14 +35,15 @@ const getConvertedProbeFromPostmanItem = (item: any) => {
     name: item.name,
     requests: [
       {
+        id: req.name,
         url: req.url.raw,
         method: req.method,
         headers: req?.header?.reduce(
           (obj: any, it: any) => Object.assign(obj, { [it.key]: it.value }),
           {}
         ),
         body: req?.body?.raw || JSON.parse('{}'),
-        timeout: 10000,
+        timeout: 10_000,
```

Comment:
What's the meaning of the underscore?

Reply:
In TS2.7, you can add separators in a number so it's easier to read. https://mariusschulz.com/blog/numeric-separators-in-typescript